banco

```diff
@@ -1,10 +1,14 @@
 case $1 in
   run)
-    docker run --rm --name esqueleto-db -p 5432:5432 -e POSTGRES_PASSWORD=esqueleto -d postgres
+    docker run --rm --name esqueleto-db -p 5432:5432 -e POSTGRES_PASSWORD=esqueleto -d postgres 
```

Comment:
Dá para remover a flag `--rm`, assim, quando o `docker stop` for executado, o container não é deletado e, consequentemente, os dados são preservados.